src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 10px;
```

Comment:
Padding should be stored in a variable in order to access it when calculating the width.